decode_morse.rb

```diff
@@ -0,0 +1,50 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(morse_code)
+  MORSE_CODE[morse_code]
+end
+
+def decode_word(morse_code_word)
+  morse_characters = morse_code_word.split
+  decoded_characters = morse_characters.map do |morse_character|
+    decode_char(morse_character)
+  end
+  decoded_characters.join
+end
+
+def decode_message(morse_code_message)
+  morse_words = morse_code_message.split
+  decoded_words = morse_words.map do |morse_word|
+    decode_word(morse_word)
+  end
+  decoded_words.join(' ')
+end
```

Comment:
- [ ] Kindly ensure the output of the code matches the give requirements

Your output
![image](https://github.com/Lornakaboro/Decode-morse-code/assets/81932478/135306a4-4825-4a08-a631-5dd819bd0c9f)

Expected output
![image](https://github.com/Lornakaboro/Decode-morse-code/assets/81932478/f705430e-93ec-436c-bfc9-a254e6a39787)